cli/src/config/misc.toml

```diff
@@ -5,9 +5,6 @@ amend = ["squash"]
 co = ["checkout"]
 unamend = ["unsquash"]
 
-[format]
-tree-level-conflicts = true
-
```

Comment:
I think we generally leave defaults in the file if possible so `jj config list --include-defaults` shows it.